test/built-ins/Promise/prototype/race/S25.4.4.3_A2.2_T3.js

```diff
@@ -3,6 +3,7 @@
 
 /*---
 info: > 
+es6id: S25.4.4.3_A2.2_T3
```

Comment:
Have to watch for the "info: >" form and put the es6id after it... looks like there are a few cases where es6id is inserted in a bad place.